tests/sparseml/export/transformers/test_generative_transformers.py

```diff
@@ -119,4 +120,21 @@ def test_export_samples(self, setup):
             target_path=target_path,
             task=task,
             num_export_samples=num_samples,
+            **dict(
+                data_args=dict(
+                    dataset_name="wikitext", dataset_config_name="wikitext-2-raw-v1"
+                )
+            ),
         )
+
+        assert (target_path / "deployment" / "model.onnx").exists()
+        assert (
+            len(os.listdir(os.path.join(target_path, "sample-inputs"))) == num_samples
+        )
+        assert (
+            len(os.listdir(os.path.join(target_path, "sample-outputs"))) == num_samples
+        )
+        assert np.load(
+            glob.glob(os.path.join(target_path, "sample-inputs/*"))[0],
+            allow_pickle=True,
+        )["arr_0"]
```

Comment:
Can we also add in a correctness test here? I guess that relies on https://github.com/neuralmagic/sparseml/pull/1938 being merged first